official/vision/beta/projects/panoptic_maskrcnn/serving/panoptic_segmentation.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Detection input and model functions for serving/inference."""
+
+import tensorflow as tf
+
+from official.vision.beta.projects.panoptic_maskrcnn.configs import panoptic_maskrcnn as cfg
+from official.vision.beta.projects.panoptic_maskrcnn.modeling import factory
+from official.vision.beta.serving import detection
+
+
+class PanopticSegmentationModule(detection.DetectionModule):
+  """Detection Module."""
+
+  def _build_model(self):
```

Comment:
can you create the model outside instead, and pass it to the export module during intialization?